elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -71,4 +71,8 @@
 	@Message(id = ES_BACKEND_MESSAGES_START_ID + 9,
 			value = "'%1$s' is not a remote analyzer. It will be ignored")
 	void analyzerIsNotRemote(String name);
+
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 10,
+			value = "Elasticsearch connection time-out; check the cluster status, it should be 'green';\n Request:\n========\n%1$sResponse:\n=========\n%2$s" )
```

Comment:
ok yes I have no strong opinion.. go ahead if you like it better.

Still, I'm sure that some people are stuck with their tools of the 90ths and will hate you as it might not be their choice to change that :)